### PR TITLE
Added awesome new prog.py file with greet function

### PR DESCRIPTION
**Pull request by Mladen-Robotics**
added function hello(name) that takes name as an argument and greets people with hello + name

### CODE REVIEW

**Comment by Mladen-Robotics** [comment]
### Here is my new **change**

[article about python](https://pythonbasics.org/why-python-is-awesome/)

File: prog.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+def greet(name):
+	print(f"Hello {name}"}
```
Comment (MartinMihalkov) [inline prog.py:2]: What is this line of code about?

**Review by MartinMihalkov — CHANGES_REQUESTED** [review]
Please add addition function

**Comment by EmPtrov** [comment]
What is this repository

**Comment by MartinMihalkov** [comment]
@EmPtrov  how are you?

**Comment by Mladen-Robotics** [comment]
Added addition function to prog.py

**Review by MartinMihalkov — APPROVED** [review]

**Comment by MartinMihalkov** [comment]
Pull request was merged.Locking conversation!

**Pull request merged**